webdriver/tests/bidi/network/add_intercept/invalid.py

```
@@ -1,3 +1,5 @@
+# META: timeout=long
+
```

I'd expect invalid arguments tests to fail fast. Are you sure it is required?

---

Removed for now.